datasets/mx/senators/mx_senators.yaml

```diff
@@ -0,0 +1,20 @@
+title: Mexico's senators
+entry_point: crawler.py
+prefix: mx-senators
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Members of the Mexican Senate
+description: |
+  This dataset includes individuals that are currently senators in Mexico.
```

Comment:
Could you expand a bit - like is the senate the entirety of mexico's national legislative branch? or one of two houses?